Makefile

```diff
@@ -68,7 +68,7 @@ assets:
 
 promu:
 	@GOOS=$(shell uname -s | tr A-Z a-z) \
-	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
```

Comment:
Because, when cross-compiling, we set the `GOOS` and `GOARCH` for the target platform. Without redefining those envvars here, `go get` would compile promu for the target platform which would not work on a `linux/amd64` based container.